packages/features/portfolio/src/side-drawers/cool-down-drawer.tsx

```diff
@@ -21,19 +21,17 @@ import {
   useAccountDefinition,
   useSelectedNetwork,
   useTransactionStatus,
-  useWalletConnectedNetwork,
 } from '@notional-finance/notionable-hooks';
 import { SNOTEWeightedPool } from '@notional-finance/core-entities';
 
 export const CoolDownDrawer = () => {
   const theme = useTheme();
   const account = useAccountDefinition(Network.mainnet);
-  const selectedNetwork = useSelectedNetwork();
   const { isReadOnlyAddress, onSubmit, transactionStatus, transactionHash } =
     useTransactionStatus(Network.mainnet);
   const { days, coolDownEnd, coolDownBegin } = useCoolDownDrawer();
-  const walletConnectedNetwork = useWalletConnectedNetwork();
```

Comment:
I'm not sure this will work, the selected network will always ben mainnet in this case, we would need to check whether or not their wallet is connected to mainnet.

Reply:
Maybe we should maintain the useWalletConnectedNetwork but just expose a function that calls onboard to get the current network inside a callback or something.

Reply:
I brought back the old hook that I updated to use the new context. Which should work fine.